modules/ocf_stats/files/munin/mail-munin-alert

```diff
@@ -33,14 +34,34 @@ IGNORED_WARNINGS = {
 
 # Enumerate uninteresting discrete values (e.g. exit codes)
 IGNORED_VALUES = {
+    # smartctl exits nonzero even if there are no current errors
     'smartctl_exit_status': {32, 64, 96, 128, 160, 192, 224},
 }
 
 
 def send_alert(host, plugin, mail_to=MAIL_ROOT):
-    subject = 'Munin alert for {}'.format(host)
-    body = ''.join(stdin)
-    body += '\nSee graph at {}/{}/{}.html'.format(MUNIN_HOSTS_URL, host, plugin)
+    ldap_entry, = hosts.hosts_by_filter('(cn={})'.format(host))
+    dns_names = ', '.join(sorted(
+        ldap_entry.get('dnsCname', []) + ldap_entry.get('dnsA', []),
+    )) or '(none)'
+
+    subject = 'Munin alert for {} ({[type]})'.format(host, ldap_entry)
+    body = (
+        '{status}\n'
+        'See graph at https://munin.ocf.berkeley.edu/ocf.berkeley.edu/{host}/{plugin}.html\n'
+        '\n'
+        'Host details:\n'
+        '  - Type: {ldap_entry[type]}\n'
+        '  - Environment: {ldap_entry[environment][0]}\n'
```

Comment:
side note, we should probably change our ldap schema so that "environment" can have only one value